components/TextFields/examples/experimental/InputChipViewExampleViewController.m

```diff
@@ -91,22 +96,14 @@ - (void)addSubviews {
   self.scrollViewSubviews = @[
     [self createToggleErrorButton],
     [self createResignFirstResponderButton],
-    [self createLabelWithText:@"High density outlined InputChipView:"],
-    [self createOutlinedNonWrappingInputChipViewWithMaximalDensity],
-    [self createLabelWithText:@"Average density outlined InputChipView:"],
-    [self createOutlinedNonWrappingInputChipView],
-    [self createLabelWithText:@"Low density outlined InputChipView:"],
-    [self createOutlinedNonWrappingInputChipViewWithMinimalDensity],
-    [self createLabelWithText:@"Wrapping outlined InputChipView:"],
-    [self createOutlinedWrappingInputChipView],
-    [self createLabelWithText:@"High density filled InputChipView:"],
-    [self createFilledNonWrappingInputChipViewWithMaximalDensity],
-    [self createLabelWithText:@"Average density filled InputChipView:"],
+    [self createLabelWithText:@"Filled InputChipView:"],
```

Comment:
Is the text of these labels part of a code that will turn into an example?  If not, shouldn't they be localized?